src/profile-logic/process-profile.js

```diff
@@ -651,7 +651,6 @@ export function processProfile(geckoProfile: GeckoProfile): Profile {
   const meta = Object.assign({}, geckoProfile.meta, {
     preprocessedProfileVersion: CURRENT_VERSION,
   });
-  delete meta.shutdownTime;
```

Comment:
Flow didn't like this so I decided to keep it because I saw no reason to remove just this property.

But now that I look at it again, I think that older processed profile will miss it. So should I upgrade the version and add something in it? And if yes, what should I put ? The highest sample time ?

Reply:
Uhh... weird. I'm not sure. I think you could satisfy flow by doing `delete (meta: Object).shudtownTime`. Or maybe create an intermediate object that isn't typed yet. I'm not sure. It does seem like a case that Flow would have issues with.

Reply:
But yeah, I think @mstange should definitely weigh in.

Reply:
I would prefer the property be removed. `shutdownTime` is something that describes a single process, but the `meta` object in the processed profile format describes the whole profile, which is made up of multiple processes. So you wouldn't know which of those processes `shutdownTime` refers to, and you'd never want to look at the property anyway because each thread has a `processShutdownTime` property that you'd use instead.

Could you list the properties individually instead, in order to make flow happy?

```js
const meta = {
  interval: geckoProfile.meta.interval,
  startTime: geckoProfile.meta.startTime,
  abi: geckoProfile.meta.abi,
  misc: geckoProfile.meta.misc,
  oscpu: geckoProfile.meta.oscpu,
  platform: geckoProfile.meta.platform,
  processType: geckoProfile.meta.processType,
  product: geckoProfile.meta.product,
  stackwalk: geckoProfile.meta.stackwalk,
  toolkit: geckoProfile.meta.toolkit,
  version: geckoProfile.meta.version,
  preprocessedProfileVersion: CURRENT_VERSION,
}
```

Reply:
Thanks, I did just this :)